css-brochure-form/styles.css

```diff
@@ -0,0 +1,97 @@
+html {
+  box-sizing: border-box;
+}
+
+form {
+  width: 800px;
+  margin: auto;
+  justify-content: center;
+}
+
+body {
+  background-color: rgb(255 255 255);
+  width: auto;
+  margin: auto;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
+}
+
+h3 {
+  color: #ec5f59;
+  text-transform: uppercase;
+  text-align: center;
+  font-size: 2em;
+}
+
+p {
+  text-transform: uppercase;
+  font-weight: 400;
+}
+
+::placeholder {
+  padding-top: 7em;
+  padding-left: 1em;
+  padding-bottom: 5em;
+  font-size: 22px;
+  color: lightgrey;
+}
+
+.input {
+  border-radius: 8px;
+  padding-top: 1em;
+  padding-bottom: 1em;
+  font-size: 19.2px;
+  width: 100%;
+  border-style: solid;
+  border-color: rgb(205 206 205);
+  border-width: 3px;
+}
+
+.button-class {
+  text-align: right;
+  margin-top: 6px;
+}
+
+.labels {
+  margin-left: 14px;
+  font-weight: 500;
+}
+
+fieldset {
+  border-style: none;
+}
+
+#button {
+  padding: 15px 40px;
+  background-color: rgb(236 94 87);
+  border-radius: 25px;
+  text-transform: uppercase;
+  color: white;
+  text-align: right;
+  border-style: none;
+  margin-top: 5px;
+  font-size: larger;
+}
+
+#hiring {
+  color: rgb(98 98 98);
+  padding-top: 15px;
+  padding-left: 10px;
+  padding-bottom: 15px;
+  border-radius: 8px;
+  width: 100%;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 400;
```

Comment:
These two lines are not necessary, they are only needed in the `@font-face` rules we provided for you.